imagemodal/views.py

```diff
@@ -0,0 +1,51 @@
+"""
+Handle view logic for the Image Modal XBlock
```

Comment:
Just makes things cleaner and easier to work with by segregating this logic out on its own..